uint16 convoystatus
uint8[4] convoyId
uint8 convoyjoinrequest
uint8 convoyleavenotice
st_CVRES[] convoyacceptresponse
